### PR TITLE
blog design for base-theme

### PR DESCRIPTION
**Pull request by metehancelik**

### CODE REVIEW

File: addons/default/visiosoft/base-theme/resources/scss/post.scss [hunk 1]
```diff
@@ -0,0 +1,152 @@
+vedat.social {
```
Comment (muammertop) [inline addons/default/visiosoft/base-theme/resources/scss/post.scss:1]: vedat?

File: addons/default/visiosoft/base-theme/resources/scss/style.scss [hunk 2]
```diff
@@ -0,0 +1,2771 @@
+@mixin flex($align, $justify, $direction) {
+  display: flex;
+  align-items: $align;
+  flex-direction: $direction;
+  justify-content: $justify;
+}
+
+@font-face {
```
Comment (muammertop) [inline addons/default/visiosoft/base-theme/resources/scss/style.scss:8]: it be easier with google-font

File: addons/default/visiosoft/base-theme/resources/scss/style.scss [hunk 3]
```diff
@@ -0,0 +1,2771 @@
...
+  @media screen and (max-width: 768px) {
+    border: none !important;
+  }
+}
+
+.height-2 {
+  height: 118px;
+}
+
+.dik-cizgi {
```
Comment (muammertop) [inline addons/default/visiosoft/base-theme/resources/scss/style.scss:335]: pls english class name

File: addons/default/visiosoft/base-theme/resources/scss/style.scss [hunk 4]
```diff
@@ -0,0 +1,2771 @@
...
+  font-style: italic;
+  src: url('{{ asset_path("visiosoft.theme.hepsiemlak::fonts/roboto/Roboto-BlackItalic.ttf") }}') format('truetype');
+}
+
+* {
+  scroll-behavior: smooth;
+  font-family: 'Roboto', sans-serif;
+}
+
+.text_1 {
```
Comment (muammertop) [inline addons/default/visiosoft/base-theme/resources/scss/style.scss:97]: where do use these classes

File: addons/default/visiosoft/base-theme/resources/scss/style.scss [hunk 5]
```diff
@@ -0,0 +1,2771 @@
...
+
+.fs-28 {
+  font-size: 28px;
+}
+
+.fs-14 {
+  font-size: 14px !important;
+}
+
+.w-fit {
```
Comment (muammertop) [inline addons/default/visiosoft/base-theme/resources/scss/style.scss:842]: It exists in post.css why do we repeat it? all custom classes can be in a single file

**Review by muammertop — CHANGES_REQUESTED** [review]

File: addons/default/visiosoft/base-theme/resources/scss/style.scss (hunk 5 above)
Comment (metehancelik) [reply addons/default/visiosoft/base-theme/resources/scss/style.scss:842]: These are already existing files in hepsiemlak repo. I just copied them in base-theme. As @fatihalp is aware of the situation, we have not much time for configuration. Correct me if I am wrong, please.

File: addons/default/visiosoft/base-theme/resources/scss/post.scss (hunk 1 above)
Comment (metehancelik) [reply addons/default/visiosoft/base-theme/resources/scss/post.scss:1]: removed

Comment (fatihalp) [inline addons/default/visiosoft/base-theme/resources/scss/post.scss:1]: vedat.social ?

@metehancelik

Comment (metehancelik) [reply addons/default/visiosoft/base-theme/resources/scss/post.scss]: unnecessary class, deleted

**Comment by metehancelik** [comment]
@muammertop @diashalabi can you please check my last commits?

File: addons/default/visiosoft/base-theme/resources/views/addons/anomaly/posts-module/posts/articles.twig [hunk 6]
```diff
@@ -0,0 +1,35 @@
+<div class="container mt-2 mb-4 px-0">
+    <div class="d-flex justify-content-between align-items-center mb-2">
+        <h5 class="text-left ml-2">{{ trans('visiosoft.theme.hepsiemlak::field.news_title', {name: setting_value('streams::name')}) }}</h5>
```
Comment (muammertop) [inline addons/default/visiosoft/base-theme/resources/views/addons/anomaly/posts-module/posts/articles.twig:3]: hepsiemlak?

**Review by muammertop — CHANGES_REQUESTED** [review]

**Review by muammertop — APPROVED** [review]

File: addons/default/visiosoft/base-theme/resources/views/addons/anomaly/posts-module/posts/articles.twig (hunk 6 above)
Comment (metehancelik) [reply addons/default/visiosoft/base-theme/resources/views/addons/anomaly/posts-module/posts/articles.twig]: changed to base

**Review by diashalabi — APPROVED** [review]

**Pull request merged**